fastlane/lib/fastlane/actions/sigh.rb

```diff
@@ -22,11 +23,13 @@ def self.run(values)
           Actions.lane_context[SharedValues::SIGH_PROFILE_PATH] = path # absolute path
           Actions.lane_context[SharedValues::SIGH_PROFILE_PATHS] ||= []
           Actions.lane_context[SharedValues::SIGH_PROFILE_PATHS] << path
-          Actions.lane_context[SharedValues::SIGH_UDID] = ENV["SIGH_UDID"] if ENV["SIGH_UDID"] # The UDID of the new profile
+
+          uuid = ENV["SIGH_UUID"] || ENV["SIGH_UDID"] # the UUID of the profile
```

Comment:
I feel like we are using ENV["SIGH_UUID"] to pass the uuid from Sigh:Worker.start command, opposed to respecting the Environment variable from the user.

If this is the case, I think we can just remove ENV["SIGH_UDID"] in a few places.